inception-project/inception
# #1136 - Qualifiers editable for read-only KBs

**What's in the PR**
- Show edit button only if the KB is not read only

**How to test manually**
* View a reified remote KB on the KB page and check if the button for editing qualifiers is there

**Automatic testing**

**Documentation**
